make/photon/registry/builder

```diff
@@ -22,6 +22,13 @@ cur=$PWD
 TEMP=`mktemp -d /$TMPDIR/distribution.XXXXXX`
 git clone -b $VERSION https://github.com/docker/distribution.git $TEMP
 
+# add patch 2879
```

Comment:
is it 2553 or 2879? it seems difference from the REGISTRYVERSION above.

Reply:
2879 is the pull request number,
2553 is the github issue number, we will the issue number as tag, it's better for user to get more detail and background.

Reply:
Can you explain this in the comment?  Plus, let's also clarify this is only needed for registry 271

Reply:
added in the commit-msg.